lib/rucio/core/replica_sorter.py

```diff
@@ -158,6 +158,59 @@ def __get_distance(se1, client_location, ignore_error):
     return cache_val
 
 
+custom_distance_table = {}
+custom_distance_download_time = None
+
+
+def __download_custom_distance_table() -> None:
+    global custom_distance_table, custom_distance_download_time
```

Comment:
See comment below with memcache, maybe you can do this without having to use `global`?

Reply:
I'm now using memcache instead of `global`